docs/documentation/_data/deckhouse-alerts.yml

```
@@ -906,13 +957,19 @@ alerts:
       module: control-plane-manager
       edition: ce
       description: |
-        The etcd database size in use on instance `{{ $labels.instance }}` is less than 50% of the actual allocated disk space, indicating potential fragmentation
+        The etcd database size in use on instance `{{ $labels.instance }}` is less than 50% of the allocated disk space, indicating potential fragmentation. Additionally, the total storage size exceeds 75% of the configured quota.
 
-        Possible solutions:
-        - You can do defragmentation. Use the following command:
-          `kubectl -n kube-system exec -ti etcd-{{ $labels.node }} -- /usr/bin/etcdctl --cacert /etc/kubernetes/pki/etcd/ca.crt --cert /etc/kubernetes/pki/etcd/ca.crt --key /etc/kubernetes/pki/etcd/ca.key --endpoints https://127.0.0.1:2379/ defrag --command-timeout=30s`
+        To resolve this issue, defragment the etcd database by running the following command:
+
+        ```bash
+        kubectl -n kube-system exec -ti etcd-{{ $labels.node }} -- /usr/bin/etcdctl \
+          --cacert /etc/kubernetes/pki/etcd/ca.crt \
+          --cert /etc/kubernetes/pki/etcd/ca.crt \
+          --key /etc/kubernetes/pki/etcd/ca.key \
+          --endpoints https://127.0.0.1:2379/ defrag --command-timeout=30s
+        ```
       summary: |
-        etcd database size in use is less than 50% of the actual allocated storage, indicating potential fragmentation, and the total storage size exceeds 75% of the configured quota.
+        Etcd database size in use is less than 50% of the allocated storage.
```

```suggestion
        etcd database size in use is less than 50% of the allocated storage.
```

I think we have to use it in lowercase, as the name of the project is `etcd`